docs/node/web3.md

```diff
@@ -0,0 +1,201 @@
+node configuration. By default, the Emerald and Sapphire ParaTimes prevent
+you from executing *expensive* transactions such as simulating EVM calls
+without paying for gas. If your Oasis node instance will only be used by the
+Web3 gateway(s) and otherwise not exposed to the internet, we suggest that you
+enable expensive transactions by adding the following to your Oasis node
+config:
+
+```yaml title='config.yml'
+# ... sections not relevant are omitted ...
+runtime:
+  mode: client
+  paths:
+    - <emerald_bundle_path>
+    - <sapphire_bundle_path>
+
+  config:
+    # Use the following IDs:
+    # 000000000000000000000000000000000000000000000000e2eaa99fc008f87f # Emerald on Mainnet
+    # 00000000000000000000000000000000000000000000000072c8215e60d5bca7 # Emerald on Testnet
+    # 000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c # Sapphire on Testnet
+    "<emerald_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+    "<sapphire_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+```
+
+## Download the Web3 Gateway
+
+Check the required version of the Web3 gateway based on which network you will
+be using it with ([Mainnet], [Testnet]). Next, download the Oasis-provided
+binaries from the [official github repository][github-releases].
+
+Alternatively, you can download the source release and compile it yourself.
+Consult the [README] file for more information.
+
+## Running the Gateway on Testnet/Mainnet
+
+Copy the content below to the config file of your gateway and adapt it
+accordingly to fit your needs:
+
+```yaml title='gateway.yml'
+# Uncomment the runtime_id below.
+#runtime_id: "000000000000000000000000000000000000000000000000e2eaa99fc008f87f" # Emerald on Mainnet
+#runtime_id: "00000000000000000000000000000000000000000000000072c8215e60d5bca7" # Emerald on Testnet
+#runtime_id: "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c" # Sapphire on Testnet
+# Path to your internal.sock file in the root Oasis node datadir.
+node_address: "unix:<path-to-oasis-node-unix-socket>"
+
+# By default, we index the entire blockchain history.
+# If you are low on disk space or you use the gateway just for submitting transactions, enable
+# pruning below.
+enable_pruning: false
+pruning_step: 100000
+indexing_start: 0
+
+log:
+  level: debug
+  format: json
+
+database:
+  # Change host and port, if PostgreSQL is running somewhere else.
+  host: "127.0.0.1"
+  port: 5432
+  # Enter your database name, username and password.
+  db: <postgresql_db>
+  user: <postgresql_user>
+  password: <postgresql_password>
+  dial_timeout: 5
+  read_timeout: 10
+  write_timeout: 5
+  max_open_conns: 0
+
+gateway:
+  # Uncomment the chain_id below.
+  #chain_id: 42262 # Emerald on Mainnet
+  #chain_id: 42261 # Emerald on Testnet
+  #chain_id: 23295 # Sapphire on Testnet
```

Comment:
```suggestion
  chain_id: {{ chain_id }}
```
And in the comment below the example values takes from network parameters pages.